llvm/test/CodeGen/RISCV/stack-clash-prologue-nounwind.ll

```diff
@@ -0,0 +1,208 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -mtriple=riscv64 -mattr=+m,+v -O2 < %s \
+; RUN:   | FileCheck %s -check-prefix=RV64I
```

Comment:
Why is this only testing rv64?

Reply:
@lenary hi, is there anything else I should change?